docs/learn-staking.md

```diff
@@ -74,74 +78,72 @@ There are three different accounts for managing your funds: `Stash`, `Controller
+For every session, validators will send an "I'm Online" message to indicate they are online. Failure to send the heartbeat means unresponsiveness. Depending on the repeated offences and how many other validators were offline, slashing will occur. If one-third of all validators are unresponsive, 5% of their bonded DOTs will be slashed.
 
 Here is the formula for calculation:
 
     Let x = offenders, n = total no. validators
 
     Min( (3 * (x - 1)) / n, 1) * 0.05
 
-
-Validators should have a well-architected network infrastructure to ensure the node is running to reduce the risk of being slashed, for example, having high availability support to guarantee even though when the node went offline, you can still have another machine to take place.
+Validators should have a well-architected network infrastructure to ensure the node is running to reduce the risk of being slashed. A high availability setup is desirable, preferably with backup nodes that kick in **only once the original node is verifiably offline** (to avoid double-signing and being slashed for equivocation - see below), together with proxy nodes to avoid being DDoSed when your validator node's IP address is exposed. A comprehensive guide on secure validator setup is in progress with the draft available [here](https://wiki.polkadot.network/docs/en/maintain-guides-secure-validator).
 
 ### Grandpa Equivocation
 
 A validator signed two or more votes in the same round on different chains.
 
-### Babe Equivocation 
-
-A validator produces two or more blocks on the relay chain in the same time slot. 
+### Babe Equivocation
 
+A validator produces two or more blocks on the relay chain in the same time slot.
 
 Grandpa and Babe equivocation slashing amount is calculated as below:
 
     Let x = offenders, n = total no. validators
 
     Min( (3 * x / n )^2, 1)
 
-Validators may run their nodes on multiple machines to make sure they can still perform validation work in case if one of their nodes goes down, but if they do not have a good coordination to manage those machines to do signing that can potentially cause the equivocation problem, hence, it would be better to have something like KMS (Key Management Server) as a middleware in between those machines to coordinate those tasks in order to avoid this kind of problem occurs.
+Validators may run their nodes on multiple machines to make sure they can still perform validation work in case if one of their nodes goes down. It should be noted that if they do not have a good coordination to manage those machines to do signing, the equivocation problem becomes a possibility. Hence, it would be better to have something like KMS (Key Management Server) as a middleware between those machines to coordinate.
```

Comment:
We currently don't support remote signing so we shouldn't recommend people to do it.
```suggestion
Validators may run their nodes on multiple machines to make sure they can still perform validation work in case one of their nodes goes down. It should be noted that if they do not have good coordination to manage signing machines, then equivocation is possible.
```